### DISCUSSION TITLE
有没有好心人贡献一些界面

### DISCUSSION THREAD

**Discussion by xmsz-stu** [ant-design/pro-chat#73] [General]
现在这个有点丑.. 就像我们公司随便找个前端做得一样

**Comment by ONLY-yours** [comment]
怎么说呢，你可以通过各种样式去改动它 hhh

**Comment by ONLY-yours** [comment]
不过说的倒是对的，感觉需要一个好看一点的 showCase 场景

**Comment by arvinxx** [comment]
@xmsz-stu 基础组件的目标不是为了做的花里胡哨，满足基础功能诉求和交互可用性即可，目前我们的样式是基于 antd 来的，需要自定义样式完全可以自行修改。做的太花反而不容易适兼容诸多场景。

showCase这块， 我们后面可以考虑给几个示例，比如 ChatGPT 的风格复刻 / Shadcn 风格等等 @ONLY-yours
